### PR TITLE
implement app.quit() (fixes #9)

### LINKED ISSUE #9: implement app.quit
The "hello world" app calls *app.quit* to quit the app when all windows have closed (unless Mac), so we should implement it.

### PR DESCRIPTION
**Pull request by mykmelez**

### CODE REVIEW

File: positron/modules/atom_browser_app.js [hunk 1]
```diff
@@ -4,7 +4,20 @@
 
 "use strict";
 
-exports.app = { /* stub */ };
+const Cc = Components.classes;
+const Ci = Components.interfaces;
+const Cr = Components.results;
+const Cu = Components.utils;
+
```
Comment (marcoscaceres) [inline positron/modules/atom_browser_app.js]: maybe: 
```JS
const { classes: Cc, interfaces: Ci, results: Cr, utils: Cu } = Components;
```
Also, are all those used?

File: positron/modules/atom_browser_app.js [hunk 2]
```diff
@@ -4,7 +4,20 @@
...
+Cu.import('resource://gre/modules/Services.jsm');
+
+exports.app = {
+  quit: function() {
```
Comment (marcoscaceres) [inline positron/modules/atom_browser_app.js]: You don't need function(), it's implied. I.e., just `quit(){ ... },`

File: positron/modules/atom_browser_app.js (hunk 1 above)
Comment (mykmelez) [inline positron/modules/atom_browser_app.js]: > const { classes: Cc, interfaces: Ci, results: Cr, utils: Cu } = Components;

Indeed, that's less verbose. Fixed in b3f8353.

Comment (mykmelez) [inline positron/modules/atom_browser_app.js]: > Also, are all those used?

They aren't, but I'd prefer to make this boilerplate consistent across modules that access Components.

File: positron/modules/atom_browser_app.js (hunk 2 above)
Comment (mykmelez) [inline positron/modules/atom_browser_app.js]: > You don't need function(), it's implied. I.e., just quit(){ ... },

Ah, method definitions, nifty. Fixed in 3d95433.

**Pull request merged**